### DISCUSSION TITLE
Welcome to AlphaSimR Discussions!

### DISCUSSION THREAD

**Discussion by gaynorr** [gaynorr/AlphaSimR#32] [Announcements]
## Welcome to AlphaSimR Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by LeoLBhering** [comment]
Dear Gaynorr, 
Congratulations on developing AlphaSimR, an exceptional package. I tried to find the function that produces gametes in AlphaSimR. I know it should use the distance between genes and move along the chromosomes. But where is it described?
Does it consider any interference rate between genes? Thank you.

**Reply by gregorgorjanc** [reply]
@LeoLBhering I addressed this in a separate discussion post at https://github.com/gaynorr/AlphaSimR/discussions/227 so it's more visible than this follow-up question to the general post
